### ISSUE TITLE
How to Boost Your Courage

### ISSUE THREAD

**Issue by Gradesfixer** [Gradesfixer/study#1]
Courage is a character trait everyone desires to possess. It is crucial in succeeding in many situations of life. However, many falter due to undesirable past experiences. Are you an individual and you have lost courage in your endeavors? Here are some ways to boost your courage:

1)Remind Yourself Your Worth.
![image](https://user-images.githubusercontent.com/49944782/56966830-99a4dc80-6b68-11e9-882d-344eda718af5.png)

You have powers and abilities. However, how often do you recognize and remind yourself about them? You often [concentrate](https://www.theguardian.com/money/2010/jul/24/secret-to-improving-concentration) on your fears, right? Divert your attention from your fears to your abilities. Verbalize them. Through this, you will realize how you regenerate courage, slowly but steady. Eventually, you will take actions despite the fear. 

2)Identify Your Fears.

You may have realized you have been living in denial. You hesitate to admit your fears. Write a list of them and figure out what causes them. After that, develop an action plan to overcome them. This action plan should be convenient for you. Make an effort and overcome your first fear. Then be easy to overcome the rest.
 
3)Take Risks And Accept Challenging Situations.

Stepping out of your comfort zone is vital in boosting your courage. Are you feeling frightened to change? It is normal to feel scared. However, accepting challenging tasks will help you realize your capabilities. It will help you learn more about yourself. Whenever you solve a problem, you learn a new skill, which in itself boosts your confidence. Taking a task each day, augment your courage daily. 

4)Change Your Attitude.

Attitude matters. If you have been having negative thoughts towards your situation, chances are you will be stuck. You will continue doing things that make you comfortable. In the end, there will be no positive difference. Regardless of what you are feeling, embrace the attitude of positivity. Accept failure, learn, and progress. You may need to put these feelings to use by writing essays about it in school. An essay about courage https://gradesfixer.com/free-essay-examples/courage/ demands one to voice out his or her inner feelings through narrating past experiences.

5)Stop Comparing Yourself With Others.

Accepting your unique [personality](https://en.wikipedia.org/wiki/Personality) traits may take much effort. If you compare yourself with others, you will concentrate on your weaknesses. Therefore, you will lose the urge to learn and be yourself. Stop comparing yourself with anybody. Instead, study their strengths and learn from them. 

6)Meditate.

Meditation reduces stress. It helps you recognize and accept the present situation. Through meditation, you overcome past fear and get the courage to begin each day afresh. Therefore, your past will not control your present. Meditate more often to achieve better results. 

Conclusion

To sum up, boosting courage requires daily decision and the willingness to progress. It requires you to identify your fears. Figure out their causes and how to overcome them. Take the simple steps. Most importantly, never stay in your comfort zone. It might take time, but the results will be appeasing.
